sonolark/cmd/root.go

```diff
@@ -73,19 +74,18 @@ func getRootCmd(env map[string]string) *cobra.Command {
 
 			predeclared, err := getLibraryFuncs(in.KubeConfigPath, env)
 			if err != nil {
-				return nil
+				return
```

Comment:
We returned nil here? That seems like it must have been a temporary thing that never got repaired. We obviously need to return some message here.